assets/schema/kubernetes-schema.graphql

```diff
@@ -5774,6 +5794,14 @@ input namespace_NamespaceSpec_Input {
   name: String!
 }
 
+input node_NodeDrainSpec_Input {
+  deleteEmptyDirData: Boolean
+  force: Boolean
+  gracePeriodSeconds: Int
+  ignoreAllDaemonSets: Boolean
```

Comment:
At the moment it is defaulted to true. @michaeljguarino what behaviour would you expect here?